projects/core/styles/components/link.less

```diff
@@ -59,13 +59,15 @@
         background: color-mix(in lch, currentColor 12%, transparent);
     });
 
-    @media @tui-mouse {
-        &[data-appearance='']:hover {
-            opacity: 0.7;
-        }
+    .appearance-hover({
+        text-decoration-thickness: 1px;
```

Comment:
`text-decoration-color: currentColor` instead. And also let's add it for `.appearance-active(...)` so that it works on mobile too where there's no hover state. Also please add `text-decoration-line: underline` so that regular appearance action links have underline on hover.

Reply:
fixed